### DISCUSSION TITLE
puan sıralamasına girmek için kaç puanım olmalı  puanıma nereden bakabilrim

### DISCUSSION THREAD

**Discussion by ferhat38** [IBAX-io/go-ibax#1375] [Soru Cevap - Türkçe]
arkadaşlar sıralamada il 1000 içine girmemiz gereklidir.Puan ve sıralamasına bakmanız için development sekmesinden  honor developers kısmından bakabilir siniz.

**Comment by Eymenedip** [comment]
Slm

**Comment by natolu2738** [comment]
Teşekkürler
